rootfs/Dockerfile

```diff
@@ -37,6 +37,13 @@ RUN setcap cap_net_bind_service=+ep /usr/sbin/nginx \
 RUN mkdir -p /etc/ingress-controller/ssl /etc/ingress-controller/auth \
   && chown -R www-data.www-data /etc/nginx /etc/ingress-controller
 
+RUN mkdir -p /tmp/modsecurity/audit /tmp/modsecurity/upload \
+  && chown -R www-data.www-data /tmp/modsecurity \
+  && sed -i 's|/var/log/modsec_audit.log|/tmp/modsec_audit.log|g'      /etc/nginx/modsecurity/modsecurity.conf \
+  && sed -i 's|/opt/modsecurity/var/log/debug.log|/tmp/debug.log|g'    /etc/nginx/modsecurity/modsecurity.conf \
+  && sed -i 's|/opt/modsecurity/var/upload|/tmp/modsecurity/upload|g'  /etc/nginx/modsecurity/modsecurity.conf \
+  && sed -i 's|/opt/modsecurity/var/audit|/tmp/modsecurity/audit|g'    /etc/nginx/modsecurity/modsecurity.conf
```

Comment:
Are these temporary files by definition (I have never used modsecurity)? If not can we create the modsecurity dedicated folder in user's home directory?

Reply:
All this directories are temporal